### PR TITLE
Quick fixes

### LINKED ISSUE #871: "convox instances keyroll" failure inconsistency
When a `convox instances keyroll` is run while a rack update is already in progress, the command errors out but the key is not rolled back. Because the key is not rolled back, the instances are not accessible via ssh until the next successful keyroll.

### LINKED ISSUE #1035: It should be hard to delete an app
There should be more friction in the process for deleting an app. Right now, there is no confirmation, so the risk of fat fingering something important is high.

The minimum amount of friction (more is welcome) is asking for confirmation by typing the name of an app. Maybe even have the option of deleting apps disabled on the rack level?

### PR DESCRIPTION
**Pull request by MiguelMoll**
Fixes #1035, #2123, #871

### CODE REVIEW

**Comment by MiguelMoll** [comment]
[Small triage fixes](https://trello.com/c/TzDkvXFy/2261-small-triage-fixes)

File: api/models/instances.go [hunk 1]
```diff
@@ -12,6 +12,18 @@ import (
...
+	system, err := Provider().SystemGet()
+	if err != nil {
+		return err
+	}
+
+	// only allow running and converging status through
+	switch system.Status {
+	case "running", "converging":
+	default:
+		return fmt.Errorf("unable to keyroll with rack status: %s", system.Status)
```
Comment (awsmsrc) [inline api/models/instances.go]: perhaps we should say which status they CAN keyroll in too?

**Comment by awsmsrc** [comment]
circle is failing

File: api/models/instances.go (hunk 1 above)
Comment (MiguelMoll) [inline api/models/instances.go]: Not sure I follow. The only case states the status that a keyroll can take place.

**Comment by MiguelMoll** [comment]
Circle is failing due to AWS throttling during uninstall. This change prints that error for clarity.

File: api/models/instances.go (hunk 1 above)
Comment (MiguelMoll) [inline api/models/instances.go]: After reading it 100x, think you mean the error message should state that. Still waking up...

**Comment by MiguelMoll** [comment]
App deletion prompt was canceled out by https://github.com/convox/rack/pull/2129

**Pull request closed without merging**